src/js/apiCecoAdapter.js

```diff
@@ -19,44 +13,43 @@ window.App.apiCecoAdapter = {
             .reverse();
     },
 
-    _createDateAsUTC: function (date) {
-        return new Date(
-            Date.UTC(
-                date.getFullYear(),
-                date.getMonth(),
-                date.getDate(),
-                date.getHours(),
-                date.getMinutes(),
-                date.getSeconds()
-            )
-        );
+    getBitcoinRatesForPeriod: function (period) {
+        return new Promise((resolve, reject) => {
+            chrome.runtime.sendMessage({ type: 'getBitcoinPrice', period: period }, (response) => {
```

Comment:
All `chrome.` refs won't work on Firefox. Use `window.browser` instead, see the notes in the /src/js/browser-extension-api.js file.

Reply:
Fixed in https://github.com/superKalo/crypto-tab/commit/082da0236b27441bc1996369ede7726d67d4e86b?w=1